setup.py

```
@@ -48,6 +48,14 @@ def get_version():
 
 ENTRY_POINT = 'ovos-dialog-normalizer-plugin=ovos_dialog_normalizer_plugin:DialogNormalizerTransformer'
 
+def package_files(directory):
+    paths = []
+    for (path, directories, filenames) in os.walk(directory):
```

Ruff (0.12.2)

Loop control variable `directories` not used within loop body

Rename unused `directories` to `_directories`

(B007)